queries/dune_v2/period_slippage.sql

```diff
@@ -84,42 +85,42 @@ other_transfers as (
                 and evt_tx_hash = b.tx_hash
              inner join batchwise_traders bt
                 on evt_tx_hash = bt.tx_hash
-    where b.block_time between '{{StartTime}}' and '{{EndTime}}'
-      and '0x9008d19f58aabd9ed0d60971565aa8510560ab41' in (to, from)
-      and not array_contains(traders_in, from)
-      and not array_contains(traders_out, to)
-      and from not in ( -- ETH FLOW ORDERS ARE NOT AMM TRANSFERS!
+    where b.block_time between cast('{{StartTime}}' as timestamp) and cast('{{EndTime}}' as timestamp)
+      and 0x9008d19f58aabd9ed0d60971565aa8510560ab41 in (to, "from")
+      and not contains(traders_in, "from")
+      and not contains(traders_out, to)
+      and "from" not in ( -- ETH FLOW ORDERS ARE NOT AMM TRANSFERS!
```

Comment:
To reduce the complexity, we might be able to replace that at some point as it is a very heavy function.

Reply:
Are you referring to all three set containment checks here or just one of them? The select statement on the ETH flow orders could be just hard coded since we know exactly what the two addresses are. In any case all three of these containment checks are pretty small (the cardinality of trader sets is bounded above by the number of trades in the batch which is often < 10).

Generally, I think there are much worse parts of this query when it comes to efficiency (e.g.  joining  `prices.usd` on `block_time = minute` & that hideous Cartesian product being made in "buffer_trades"). 

Would love to simplify this even further sometime soon ~ our upcoming slippage V2 will be a good place to start.